torch_em/data/segmentation_dataset.py

```diff
@@ -17,6 +17,9 @@ class SegmentationDataset(torch.utils.data.Dataset):
 
     @staticmethod
     def compute_len(shape, patch_shape):
+        if patch_shape is None:
```

Comment:
Can you do 
```python
if patch_shape is None:
   return 1
```
instead? That's much easier to read and amounts to the same.

Reply:
Oh yeah, that's simple and much more readable haha. Thanks for the hint.